pkg/scheduler/framework/types.go

```diff
@@ -72,6 +72,8 @@ const (
 	Service               GVK = "Service"
 	StorageClass          GVK = "storage.k8s.io/StorageClass"
 	CSINode               GVK = "storage.k8s.io/CSINode"
+	CSIDriver             GVK = "storage.k8s.io/CSIDriver"
+	CSIStorageCapacity    GVK = "storage.k8s.io/CSIStorageCapacity"
```

Comment:
we don't have event handlers for those in https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/eventhandlers.go

Reply:
fixed in the new commit